bin/resource_management/url_database.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import grequests
+import fileinput
+from datetime import datetime
+import json
+import csv
+
+"""
+Fetches URL's from metadata and tests URL status in parallel using grequests library.
+Outputs status to 'url_database.csv'
+
+Arguments:
+fileinput - metadata file to be read ('od-do-canada.jsonl')
+batch_size - maximum number of URL's to test in parallel
+"""
+
+batch_size = int(sys.argv[2])
+
+i = 1
+prev_i = 1
+urls = []
+batch_urls = []
+responses = []
+date = []
+
+
+print("Starting...")
+print("Reading and testing URL's")
+
+for dataset in fileinput.input():

```

Comment:
`for i, dataset in enumerate(fileinput.input(), 1):`
then you don't need to initialize and update the line counter yourself